tri.html

```diff
@@ -22,41 +22,40 @@ <h1>The <span id="titleDisplay">Triangle</span> Game</h1>
 
       <div class="row" id="row-1a"></div>
 
-          <div class="row" id="row-1">
-        <div class="circle" id="one">1</div>
+      <div class="row" id="row-1">
+        <div class="circle" id="1">1</div>
```

Comment:
If you're trying to encode the index in HTML, I'd use the [data attribute](https://www.w3schools.com/tags/att_global_data.asp) instead of the ID.